services/provider/server/server.go

```diff
@@ -11,14 +11,15 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
-	"k8s.io/utils/ptr"
 	"math"
 	"net"
 	"slices"
 	"strconv"
 	"strings"
 	"time"
 
+	"k8s.io/utils/ptr"
+
```

Comment:
Is this additional new line added intentionally?

Reply:
Not an additional line but a go linter change